options/locale/locale_en-US.ini

```
@@ -2009,7 +2009,8 @@ settings.mirror_settings.push_mirror.add = Add Push Mirror
 settings.mirror_settings.push_mirror.edit_sync_time = Edit mirror sync interval
 
 settings.sync_mirror = Synchronize Now
-settings.mirror_sync_in_progress = Mirror synchronization is in progress. Check back in a minute.
+settings.pull_mirror_sync_in_progress = Syncing changes from the remote %s is currently in progress.
```

Alternatively,
```suggestion
settings.pull_mirror_sync_in_progress = Pulling changes from the pull mirror %s at the moment.
```
or
```suggestion
settings.pull_mirror_sync_in_progress = Pulling changes from the origin %s at the moment.
```

---

Pulling changes from the remote %s at the moment.